razorpay-quick-payments.php

```diff
@@ -135,37 +135,44 @@ function razorpayOrderCreationResponse()
 
                 $metadata = get_post_meta($pageID);
 
-                $amount = (int) ($metadata['amount'][0]) * 100;
+                if (empty($metadata) === true)
+                {
+                    $razorpayArgs['error'] = 'Post with given page ID not found';
+                }
+                else
+                {
+                    $amount = (int) ($metadata['amount'][0]) * 100;
 
-                $productInfo = $this->getProductDecription($metadata, $pageID);
+                    $productInfo = $this->getProductDecription($metadata, $pageID);
 
-                $name = $this->getProductName($metadata);
+                    $name = $this->getProductName($metadata);
 
-                $api = new Api($this->keyID, $this->keySecret);
+                    $api = new Api($this->keyID, $this->keySecret);
 
-                // Calls the helper function to create order data
-                $data = $this->getOrderCreationData($orderID, $amount);
+                    // Calls the helper function to create order data
+                    $data = $this->getOrderCreationData($orderID, $amount);
 
-                try
-                {
-                    $razorpayOrder = $api->order->create($data);
-                }
-                catch (Exception $e)
-                {
-                    echo 'Wordpress Error : ' . $e->getMessage();
-                }
+                    try
+                    {
+                        $razorpayOrder = $api->order->create($data);
+                    }
+                    catch (Exception $e)
+                    {
+                        echo 'Wordpress Error : ' . $e->getMessage();
```

Comment:
Shouldn't this be jsonencoded as `error` as well, now that we have a error format?